src/chapters/razor-views/exercises.rst

```diff
@@ -0,0 +1,51 @@
+Exercises: Views
+================
+
+In this chapter, we started working on an application for tracking various
+coding events around town.
+
+Open up your ``CodingEvents`` project in Visual Studio.
```

Comment:
I hear you as far as workflow but since we don't care what their repo looks like before this point, I think its a simpler ask to have them make any changes to get to a similar starting place on the same branch that they will complete the exercises on